### PR TITLE
M14 Release Notes

### PR DESCRIPTION
**Pull request by Clintonio**

### CODE REVIEW

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -3,12 +3,27 @@ Release notes
...
-----------
+Milestone 14
+------------
+
+This release continues with the goal to improve API stability and developer friendliness. There have also been more
+bug fixes and other improvements across the board.
+
+It is now possible to specify multiple IP addresses and legal identities for a single node, allowing node operators
+more flexibility in setting up nodes.
+
```
Comment (shamsasari) [inline docs/source/release-notes.rst]: We've now begun the process of demarcating which classes are part of our public API and which ones are internal. Everything found in ``net.corda.core.internal`` and other packages in the ``net.corda`` namespace which has ``.internal`` in it are considered internal and not for public use. In a future release any CorDapp using these packages will fail to load, and when we migrate to Jigsaw these will not be exported.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -4,8 +4,8 @@ Changelog
 Here are brief summaries of what's changed between each snapshot release. This includes guidance on how to upgrade code
 from the previous milestone release.
 
-UNRELEASED
-----------
+Milestone 14
+------------
```
Comment (gendal) [inline docs/source/changelog.rst]: i can't put comments against the lines... but...

line 41:  "now has can be" ??

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -57,6 +57,27 @@ UNRELEASED
...
+    * FIX inconsistent behaviour: Vault Query defaults to UNCONSUMED in all QueryCriteria types
+
+    * FIX serialization error: Vault Query over RPC when using custom attributes using VaultCustomQueryCriteria.
+
+    * Aggregate function support: extended VaultCustomQueryCriteria and associated DSL to enable specification of
+    Aggregate Functions (sum, max, min, avg, count) with, optional, group by clauses and sorting (on calculated aggregate)
+
+    * Pagination simplification
+    Pagination continues to be optional, but with following changes:
+      - If no PageSpecification provided then a maximum of MAX_PAGE_SIZE (200) results will be returned, otherwise we fail-fast with a VaultQueryExceptionto alert the API user to the need to specify a PageSpecification.
```
Comment (gendal) [inline docs/source/changelog.rst]: VaultQueryExceptionto << space

File: docs/source/release-notes.rst [hunk 4]
```diff
@@ -3,12 +3,27 @@ Release notes
...
 
-Unreleased
-----------
+Milestone 14
+------------
+
+This release continues with the goal to improve API stability and developer friendliness. There have also been more
+bug fixes and other improvements across the board.
+
+It is now possible to specify multiple IP addresses and legal identities for a single node, allowing node operators
```
Comment (gendal) [inline docs/source/release-notes.rst]: that's a big statement... did we mean for it to be so big?  A single node can represent multiple legal identities?  really?!

File: docs/source/release-notes.rst [hunk 5]
```diff
@@ -3,12 +3,27 @@ Release notes
...
+more flexibility in setting up nodes.
+
+A format has been introduced for CorDapp JARs that standardises the contents of CorDapps across nodes. This new format
+now requires CorDapps now contain their own external dependencies. This paves the way for significantly improved
```
Comment (gendal) [inline docs/source/release-notes.rst]: only need one "now"

File: docs/source/release-notes.rst [hunk 6]
```diff
@@ -3,12 +3,27 @@ Release notes
...
+dependency management for CorDapps with the release of Jigsaw. For those using non-gradle build systems it is important
```
Comment (gendal) [inline docs/source/release-notes.rst]: do we need to qualify Jigsaw for those not paying close attention to the Java development process?

File: docs/source/release-notes.rst [hunk 7]
```diff
@@ -3,12 +3,27 @@ Release notes
...
+to read :doc:`cordapp-build-systems` to learn more. For those using our ``cordformation`` plugin simply need to update
```
Comment (gendal) [inline docs/source/release-notes.rst]: Lose the "for" at the start of the sentence

File: docs/source/release-notes.rst [hunk 8]
```diff
@@ -3,12 +3,27 @@ Release notes
...
+to the latest version (``0.14.0``) to get the fixes.
 
 The transaction finalisation flow (``FinalityFlow``) has had hooks for alternative implementations, for example in
```
Comment (gendal) [inline docs/source/release-notes.rst]: has had hooks _added_?

File: docs/source/release-notes.rst [hunk 9]
```diff
@@ -33,7 +33,10 @@ scenarios where no single participant in a transaction is aware of the well know
 DemoBench has a fix for a rare but inconvenient crash that can occur when sharing your display across multiple devices,
 e.g. a projector while performing demonstrations in front of an audience.
 
-Upgrading to this release is strongly recommended in order to keep up with the API changes, removal and additions. 
+Guava types are being removed because Guava does not have backwards compatibility across versions, which has serious
```
Comment (shamsasari) [inline docs/source/release-notes.rst]: Can this be more specific: Our public API is no longer making use of Guava types due to the library not always maintaining backwards compatibility, etc, etc.

**Pull request merged**